elements/cc-rclone/package-installs.yaml

```diff
@@ -0,0 +1,9 @@
+---
+# we install rclone via the package manager
+# to get all the dependences, however, we will
+# upgrade it directly from rclone's site to get
+# latest version
+rclone:
```

Comment:
What dependencies do we miss if we just fetch a tarball from rclone's github releases?
I'm concerned that mixing repo install and script-based install could cause issues, but rclone does provide deb and rpm packages in their releases.